.github/workflows/build.yml

```diff
@@ -48,26 +48,24 @@ jobs:
         uses: ilammy/msvc-dev-cmd@v1
       - name: Build
         run: |
-          cd build\/${{ runner.os }}\${{ matrix.compiler }}
+          mkdir build
```

Comment:
Windows CI made problems here when not creating the directory. The makefiles work on a local machine when the directory is not present.
Please make sure that this also works for you then I would say to `mkdir build` here as a workaround.

Reply:
You can keep it for now, later we will move it to `dist`, or move all output to `build`. Whatever will save us time.